src/event.rs

```diff
@@ -388,55 +395,32 @@ impl<N: Notify> Processor<N> {
         }
     }
 
-    /// Process events. When `wait_for_event` is set, this method is guaranteed
-    /// to process at least one event.
-    pub fn process_events<'a>(
```

Comment:
i moved a lot of the tricky logic from here out into the main display loop in `main.rs`